### PR TITLE
docs: add explanation how an Excel file for list creation must be structured (DEV-533)

### PR DESCRIPTION
**Pull request by jnussbaum**
resolves DEV-533

### CODE REVIEW

File: docs/dsp-tools-excel.md [hunk 1]
```diff
@@ -74,72 +74,82 @@ For further information about properties, see [here](./dsp-tools-create-ontologi
...
 The output is written to the file `list.json`.
 
 ```bash
-dsp-tools excel lists list.json
+dsp-tools excel listfolder list.json
 ```
 
-The two Excel files `liste_de.xlsx` and `list_en.xlsx` are located in a folder called `lists`. `liste_de.xlsx` 
-contains German labels for the list, `list_en.xlsx` contains the English labels.
+The two Excel files `Beschreibung_de.xlsx` and `description_en.xlsx` are located in a folder called `lists`.
```
Comment (irinaschubert) [inline docs/dsp-tools-excel.md:110]:
```suggestion
The two Excel files `Beschreibung_de.xlsx` and `description_en.xlsx` are located in a folder called `listfolder`.
```

File: docs/dsp-tools-excel.md [hunk 2]
```diff
@@ -74,72 +74,82 @@ For further information about properties, see [here](./dsp-tools-create-ontologi
...
       "labels": {
-        "de": "Grobsand",
-        "en": "coarse sand"
-      }
+        "de": "erste Unterliste",
+        "en": "first sublist"
+      },
+      "nodes": [
+        {
+          "name": "first-subnode",
```
Comment (irinaschubert) [inline docs/dsp-tools-excel.md:140]:
```suggestion
          "name": "first-node",
```

File: docs/dsp-tools-excel.md [hunk 3]
```diff
@@ -74,72 +74,82 @@ For further information about properties, see [here](./dsp-tools-create-ontologi
...
+          "labels": {
+            "de": "erster Listenknoten",
+            "en": "first subnode"
```
Comment (irinaschubert) [inline docs/dsp-tools-excel.md:143]:
```suggestion
            "en": "first node of the list"
```

File: docs/dsp-tools-excel.md [hunk 4]
```diff
@@ -74,72 +74,82 @@ For further information about properties, see [here](./dsp-tools-create-ontologi
...
-    {
-      "name": "medium-sand",
-      "labels": {
-        "de": "Mittelsand",
-        "en": "medium sand"
-      }
-    },
-    {
-      "name": "coarse-sand",
+      "name": "first-sublist",
```
Comment (irinaschubert) [inline docs/dsp-tools-excel.md:133]: In my view, it is confusing to use the term "sub-list". The node called "first-sublist" here is the actual list. I would suggest using "list" or "my-list" or something similar.

**Review by irinaschubert — APPROVED** [review]
I added some suggestions for improvements, but feel free to chose whatever you find the best option.

File: docs/dsp-tools-excel.md (hunk 4 above)
Comment (jnussbaum) [reply docs/dsp-tools-excel.md:133]: As I understand, the contents of Excel column A are already nodes/sublists. The top-level list name is defined by the file name.  
In every case, my test data behaves that way...

**Pull request merged**